deploy/distrib/src/main/resources/README.TXT

```diff
@@ -56,12 +58,16 @@ Contents of this package:
 |   |   `-- lib
 |   |       `-- *.jar                    				Common libs that can be shared by several webapps (H2 listener)
 |   `-- welcome-content
-|       `-- favicon.ico
-|    
+|   |   `-- favicon.ico
+|   `-- bonita-start(.sh|.bat)                          Script used to start Bonita BPM Tomcat bundle
+|
 |-- platform-setup							            Platform creation/initialization files (Database scripts + configuration files)
 |-- license                                 			License files
 `-- README.txt                              			You are here!
 
 In order to install bonita you need to copy into the root folder of your application server:
 - the content of the directory matching it
-- for Jboss update the file bonita-platform-community.properties and bonita-tenant-community.properties to remove every occurrence of "java:comp/env/"
+- for Jboss update the files `platform-setup/platform_conf/initial/platform_engine/bonita-platform-community-custom.properties`
+  and `platform-setup/platform_conf/initial/tenant_template_engine/bonita-tenant-community-custom.properties`:
+      - uncomment lines containing "java:comp/env/"
+      - remove every occurrence of "java:comp/env/" (leave the rest of the value)
```

Comment:
still quite unclear, but i finally understood what to do. 

thoses lines may be alreay present in  with specific comment like "# JBoss configuration: uncomment lines below to configure jndi"

due to build process, they should be commented in bonita-platform-community.properties / platform-setup/platform_conf/initial/tenant_template_engine/bonita-tenant-community.properties